scripts/diff-deployed-dssspell.sh

```diff
@@ -2,23 +2,31 @@
 
 set -e
 
+command -v jq &> /dev/null || { echo '`jq` command could not be found. Please install it: https://github.com/stedolan/jq'; exit 1; }
+[[ "$ETHERSCAN_API_KEY" ]] || { echo -e "Please set ETHERSCAN_API_KEY"; exit 1; }
+
 if [[ "$1" =~  ^0x[0-9a-fA-F]{40}$ ]]; then
     deployed_spell_address=$1
 else
     # Read contract address from config.sol
-    deployed_spell_address=$(grep -oE 'deployed_spell:\s+address\((0x[a-fA-F0-9]+)\)' "src/test/config.sol" | sed -E 's/^.*\((.*)\)/\1/')
+    deployed_spell_address=$(grep -oE 'deployed_spell:\s*(address\((0x[a-fA-F0-9]{40})\)|(0x[a-fA-F0-9]{40}))' "src/test/config.sol" | sed -E 's/.*(0x[a-fA-F0-9]{40}).*/\1/')
```

Comment:
Nice catch on the new `|` operator in `address\((0x[a-fA-F0-9]{40})\)|(0x[a-fA-F0-9]{40})`, although there was just one spell like this 🙂